metadata.rb

```diff
@@ -1,13 +1,13 @@
 name             'ufw'
-maintainer       'Chef Software, Inc.'
-maintainer_email 'cookbooks@chef.io'
+maintainer       'Sous Chefs'
+maintainer_email 'help@sous-chefs.org'
 license          'Apache-2.0'
 description      'Installs and configures Uncomplicated Firewall (ufw)'
 version          '3.2.14'
 depends          'firewall', '>= 2.0'
+source_url       'https://github.com/sous-chefs/ufw'
+issues_url       'https://github.com/sous-chefs/ufw/issues'
 
 supports 'ubuntu'
-
-source_url 'https://github.com/chef-cookbooks/ufw'
-issues_url 'https://github.com/chef-cookbooks/ufw/issues'
-chef_version '>= 12.15'
+supports 'debian'
+chef_version '>= 15.3'
```

Comment:
Is this just standard that we are only supporting 15.3+? I know this is needed if the cookbook is using modern resources.

Reply:
Standard pattern at this point. According to sous-chefs.org we should be requiring 16+ but we're definitely not going to need anything from that today.